packages/core/app-layout/src/components/aboutSection/AppAboutSection.vue

```diff
@@ -0,0 +1,241 @@
+        data-testid="about-section-title"
+      >
+        {{ title }}
+      </span>
+    </template>
+
+    <template
+      v-if="$slots.actions || created || modified"
+      #actions
+    >
+      <div class="about-section-header-end">
+        <div
+          v-if="created || modified"
+          class="about-section-timestamps"
+          :class="{ 'has-actions': $slots.actions }"
+        >
+          <span
+            v-if="created"
+            class="about-section-timestamps-created"
+            data-testid="about-section-timestamps-created"
+          >{{ createdLabel }}: {{ created }}</span>
+          <span
+            v-if="created && displayModified"
+            class="about-section-timestamps-arrow"
+          >-></span>
+          <span
+            v-if="displayModified"
+            class="about-section-timestamps-modified"
+            data-testid="about-section-timestamps-modified"
+          >{{ modifiedLabel }}: {{ modified }}</span>
+        </div>
+
+        <div
+          v-if="$slots.actions"
+          class="about-section-actions"
+          data-testid="about-section-actions"
+        >
+          <slot name="actions" />
+        </div>
+      </div>
+    </template>
+
+    <template #body>
+      <div
+        v-if="isLoading"
+        data-testid="about-section-loading-skeleton"
+      >
+        <KSkeletonBox
+          height="2"
+          width="100"
+        />
+        <KSkeletonBox
+          height="2"
+          width="100"
+        />
+      </div>
+
+      <div v-else>
+        <p
+          v-if="description"
+          class="about-section-description"
+          data-testid="about-section-description"
+        >
+          {{ description }}
+        </p>
+
+        <div
+          v-if="$slots.default"
+          class="about-section-content"
+          data-testid="about-section-content"
+        >
+          <slot name="default" />
+        </div>
+
+        <hr
+          v-if="$slots['divider-section']"
+          data-testid="about-divider-section-separator"
+        >
+
+        <div
+          v-if="$slots['divider-section']"
+          class="about-divider-section"
+          data-testid="about-divider-section"
+        >
+          <slot name="divider-section" />
+        </div>
+      </div>
+    </template>
+  </KCard>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+
+const props = defineProps({
+  title: {
+    type: String,
+    default: '',
+  },
+  description: {
+    type: String,
+    default: '',
+  },
+  created: {
+    type: String,
+    default: '',
+  },
+  createdLabel: {
+    type: String,
+    default: 'Created',
+  },
+  modified: {
+    type: String,
+    default: '',
+  },
+  modifiedLabel: {
+    type: String,
+    default: 'Modified',
+  },
+  isLoading: {
+    type: Boolean,
+    default: false,
+  },
+})
+
+const displayModified = computed(() => {
+  return !!props.modified && props.modified !== props.created
```

Comment:
suggestion: if you add a check here that `props.created` exists, you can also change the `v-if="created && displayModified"` to just `v-if="displayModified"` in the template above

Reply:
Design team mentioned they may decide to only show `modified` on some pages, so I may need `displayModified` to be `true` when `props.created` doesn't exist.